src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -451,6 +447,15 @@ class SignUpModalController {
     })
   }
 
+  showVerificationCodeField () {
+    // The verification code field is only shown when the button link "Enter a verification code instead" is clicked.
+    // This makes the process of creating an account more streamlined as we remove that click.
+    const verificationCodeButtonLink = document.querySelector('.button-link.enter-auth-code-instead-link')
+    if (verificationCodeButtonLink) {
+      verificationCodeButtonLink.click();
+    }
```

Comment:
I don't think an `if` is bad, but you could also write that like this:

```suggestion
    verificationCodeButtonLink?.click();
```